core/src/test/java/com/airwallex/airskiff/common/GcsDailyLocationTest.java

```diff
@@ -1,18 +1,39 @@
 package com.airwallex.airskiff.common;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
 import com.google.api.gax.paging.Page;
+import com.google.auth.oauth2.GoogleCredentials;
 import com.google.cloud.storage.Blob;
 import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import com.google.common.collect.Lists;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
-import org.junit.jupiter.api.Test;
-import org.mockito.Mockito;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class GcsDailyLocationTest {
 
+  @Disabled
```

Comment:
let's remove this test

Reply:
Done & reverted the commit history as well.